samples/test/quickstart.js

```diff
@@ -19,32 +19,30 @@
 
 const path = require('path');
 const cp = require('child_process');
-const {before, describe, it} = require('mocha');
+const {describe, it} = require('mocha');
 // eslint-disable-next-line node/no-missing-require
-const {ServiceControllerClient} = require('');
 // eslint-disable-next-line no-unused-vars, node/no-missing-require
 const {assert} = require('chai');
 
 const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
 
 const cwd = path.join(__dirname, '..');
 
-const client = new {ServiceControllerClient}();
-
 describe('Quickstart', () => {
-  //TODO: remove this if not using the projectId
-  // eslint-disable-next-line no-unused-vars
-  let projectId;
-
-  before(async () => {
-    // eslint-disable-next-line no-unused-vars
-    projectId = await client.getProjectId();
-  });
-
   it('should run quickstart', async () => {
-    //TODO: remove this line
-    // eslint-disable-next-line no-unused-vars
-    const stdout = execSync('node ./quickstart.js', {cwd});
-    //assert(stdout, stdout !== null);
+    // We should get an error, since we don't have a DNS service
```

Comment:
Is it possible for us to create a service in the UX of long-door-651, and then test against that?